.claude/skills/ghflow-issue-expander/scripts/create_issue.sh

```
@@ -0,0 +1,41 @@
+
+create_github_issue() {
+    local title="$1"
+    local body="$2"
+
+    if [ -z "$title" ]; then
+        echo "Error: Issue title required" >&2
+        return 1
+    fi
+
+    # Create issue and capture the output
+    local output=$(gh issue create --title "$title" --body "$body" 2>&1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)